pkg/tsdbctl/tsdbctl.go

```diff
@@ -53,23 +53,34 @@ func NewRootCommandeer() *RootCommandeer {
 	commandeer := &RootCommandeer{}
 
 	cmd := &cobra.Command{
-		Use:          "tsdbctl [command]",
-		Short:        "V3IO TSDB command-line interface",
+		Use:          "tsdbctl [command] [arguments] [flags]",
+		Short:        "V3IO TSDB command-line interface (CLI)",
 		SilenceUsage: true,
 	}
 
 	defaultV3ioServer := os.Getenv("V3IO_SERVICE_URL")
 
-	cmd.PersistentFlags().StringVarP(&commandeer.verbose, "verbose", "v", "", "Verbose output")
-	cmd.PersistentFlags().Lookup("verbose").NoOptDefVal = "debug"
-	cmd.PersistentFlags().StringVarP(&commandeer.dbPath, "table-path", "t", "", "sub path for the TSDB, inside the container")
-	cmd.PersistentFlags().StringVarP(&commandeer.v3ioPath, "server", "s", defaultV3ioServer, "V3IO Service URL - ip:port")
-	cmd.PersistentFlags().StringVarP(&commandeer.cfgFilePath, "config", "g", "", "path to yaml config file")
-	cmd.PersistentFlags().StringVarP(&commandeer.container, "container", "c", "", "container to use")
-	cmd.PersistentFlags().StringVarP(&commandeer.username, "username", "u", "", "user name")
-	cmd.PersistentFlags().StringVarP(&commandeer.password, "password", "p", "", "password")
-
-	// add children
+	cmd.PersistentFlags().StringVarP(&commandeer.verbose,
```

Comment:
`verbose` was changed to `logLevel`

Reply:
MERGED `development` with my previous PR changes (including using `DefaultLoggingLevel`, which I had defined in *config.go* — info in comment https://github.com/v3io/v3io-tsdb/pull/148#issuecomment-426176080). I edited the flag's help text in *tsdbctl.go* and the related configuration-file `logLevel` key code coment *config.go* (including to remove the referene to`info` being the default logging level, as we discussed on Slack). See commit 2d22ad7.